src/index/IndexImpl.h

```diff
@@ -631,6 +632,42 @@ class IndexImpl {
   void writeConfiguration() const;
   void readConfiguration();
 
+  // Assigns the entries of the enum `TurtleParserIntegerOverflowBehavior` to
+  // their string representation.
+  inline static const ad_utility::HashMap<std::string,
+                                          TurtleParserIntegerOverflowBehavior>
+      turtleParserIntegerOverflowBehaviorMap_{
```

Comment:
This can be more readable if you assign it by an immediately invoked lambda inside ow which you use `using enum TurtleParserIntegerOverflowBehaviior`.

Reply:
Done, see [c366a3f](https://github.com/ad-freiburg/qlever/pull/1041/commits/c366a3f1f42f3f28e058c7ba60a7fdc3ae029548).